wgpu/src/lib.rs

```diff
@@ -5143,6 +5145,20 @@ pub enum Error {
         /// Description of the validation error.
         description: String,
     },
+    /// Internal GPU error, used for signaling failures in the implementation of WebGPU or the
+    /// system
```

Comment:
```suggestion
    /// Internal error. Used for signalling any failures not explicitly expected by WebGPU.
    ///
    /// These could be due to internal implementation or system limits being reached.
```

Reply:
Changed my own `fixup!` phrasing to match this with f97183d19.